pkg/settings/setting.go

```diff
@@ -18,6 +18,7 @@ import (
 )
 
 const RancherVersionDev = "2.8.99"
+const DefaultMaxUIPluginFileSizeInBytes = 20 * 1024 * 1024 // 20MB
```

Comment:
nit: can this be moved to the plugin package?

Reply:
That would cause an import cycle